src/pybind/mgr/dashboard/frontend/src/openattic-theme.scss

```diff
@@ -30,11 +30,6 @@
   CSS Fix
 */
 
```

Comment:
I think the colors in this file should also be extracted.

Reply:
Ack